core/tests/webdriverio/learnerFlow.js

```diff
@@ -130,7 +130,8 @@ describe('Learner dashboard functionality', function() {
           true
         );
       }
-      await users.logout();
+      await general.callFunctionAndCollectFullStackTraceOnError(
+        users.logout, new Error().stack);
```

Comment:
Can you add a comment above the changes made referencing the issue and explaining that it was added to debug it?

Reply:
done.